src/PIL/WebPImagePlugin.py

```diff
@@ -38,6 +38,8 @@ class WebPImageFile(ImageFile.ImageFile):
 
     format = "WEBP"
     format_description = "WebP image"
+    __loaded = -1
+    __logical_frame = -1
```

Comment:
```suggestion
    __logical_frame = 0
```

I suggest that this be zero. With -1, if I call `tell()` after unpickling,

```python
from PIL import Image
import pickle

image = Image.open('Tests/images/hopper.webp')
unpickled_image = pickle.loads(pickle.dumps(image))
print(unpickled_image.tell())
```

`tell()` reports -1, which we would consider to be invalid.

Reply:
Good idea. This surfaces another bug:

```
    def test_pickle_image(tmp_path, test_file, test_mode):
        # Act / Assert
        for protocol in range(0, pickle.HIGHEST_PROTOCOL + 1):
>           helper_pickle_string(pickle, protocol, test_file, test_mode)

Tests/test_pickle.py:65:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Tests/test_pickle.py:40: in helper_pickle_string
    assert im == loaded_im
src/PIL/Image.py:651: in __eq__
    and self.getpalette() == other.getpalette()
src/PIL/Image.py:1324: in getpalette
    self.load()
src/PIL/WebPImagePlugin.py:160: in load
    self._seek(self.__logical_frame)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <PIL.WebPImagePlugin.WebPImageFile image mode=RGB size=128x128 at 0x112A0C130>, frame = 0

    def _seek(self, frame):
>       if self.__physical_frame == frame:
E       AttributeError: 'WebPImageFile' object has no attribute '_WebPImageFile__physical_frame'

src/PIL/WebPImagePlugin.py:150: AttributeError
```


Adding `__physical_frame = 0` shows another:

```
    def test_pickle_image(tmp_path, test_file, test_mode):
        # Act / Assert
        for protocol in range(0, pickle.HIGHEST_PROTOCOL + 1):
>           helper_pickle_string(pickle, protocol, test_file, test_mode)

Tests/test_pickle.py:65:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Tests/test_pickle.py:40: in helper_pickle_string
    assert im == loaded_im
src/PIL/Image.py:651: in __eq__
    and self.getpalette() == other.getpalette()
src/PIL/Image.py:1324: in getpalette
    self.load()
src/PIL/WebPImagePlugin.py:163: in load
    data, timestamp, duration = self._get_next()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <PIL.WebPImagePlugin.WebPImageFile image mode=RGB size=128x128 at 0x10CF00100>

    def _get_next(self):
        # Get next frame
>       ret = self._decoder.get_next()
E       AttributeError: 'WebPImageFile' object has no attribute '_decoder'

src/PIL/WebPImagePlugin.py:131: AttributeError
```

Will push these to demonstrate on the CI, but do we need to call `_open` somewhere when unpickling?